tasks/main.yml

```diff
@@ -17,23 +17,24 @@
 
 - name: Install ansible tower
   include_tasks: install.yml
-  when: not restore and not backup
+  when: backup_name is not defined and not backup
```

Comment:
This condition seems odd. Maybe we can register a var called ansible_tower_install: true in a previous logic task where we are just determining what action we are about to perform (backup, restore, upgrade, install, etc. Then in this task we can just use that var as the condition.